packages/app/src/utils/get-current-page-scope.ts

```diff
@@ -4,6 +4,14 @@ import { NextRouter } from 'next/router';
 /**
  * Returns a typed DataScope based on the current route's path
  */
-export function getCurrentPageScope(router: NextRouter): RouterDataScopeKey | undefined {
-  return router.pathname.startsWith('/landelijk') ? 'nl' : router.pathname.startsWith('/gemeente') ? 'gm' : undefined;
+export function getCurrentPageScope(router: NextRouter): (RouterDataScopeKey | 'general') | undefined {
+  const { pathname } = router;
+
+  const pageScopeMap: Record<string, RouterDataScopeKey | 'general'> = {
+    landelijk: 'nl',
+    gemeente: 'gm',
+    verantwoording: 'general',
+  };
+
+  return pageScopeMap[Object.keys(pageScopeMap).find((key) => pathname.startsWith(`/${key}`)) as keyof typeof pageScopeMap] || undefined;
```

Comment:
`Object.keys(pageScopeMap).find(..) as keyof typeof pageScopeMap` is a very elaborate way of claiming this is probably not `undefined`.
And then putting it in `pageScopeMap[...] || undefined` is... peculiar.

The `keyof typeof pageScopeMap` is `string` according to line 10. Using `Array.find()` makes it `string | undefined`.

So extracting the key as separate value and then catching the `undefined` scenario should take care of that:
```suggestion
  const pageScopeMapKey = Object.keys(pageScopeMap).find((key) => pathname.startsWith(`/${key}`));
  return typeof pageScopeMapKey === 'undefined' ? undefined : pageScopeMap[pageScopeMapKey];
```